pytorch_blade/src/compiler/tensorrt/bridge/tensorrt_onnx_parser.cpp

```diff
@@ -183,74 +176,5 @@ bool TensorrtOnnxParser::IsSupportedModel(const std::string& proto_bytes) {
   return GetSegmentList(proto_bytes, nodes_segment_list);
 }
 
-bool TensorrtOnnxParser::setDynamicRange(
```

Comment:
What is the difference between `setDynamicRange` directly and given calibration data? Could you please claim the decision choice you made?

Reply:
Have added some notes about the two ways of trt quantization